### ISSUE TITLE
Some accessibility issues

### ISSUE THREAD

**Issue by xi** [okfde/offenegesetze.de#30]
There are some accessibility issue that should be easy to fix (I did not do a full test, so there may be much more):

- Links are not underlined (e.g. here: https://offenegesetze.de/faq)
- Insufficient contrast (e.g. [on gradient](https://contrast-ratio.com/#white-on-%2300e7eb) or [in the footer](https://contrast-ratio.com/#%23b5b5b5-on-%23fafafa))

**Comment by jfilter** [comment]
Thanks for pointing this out. I fixed those two issues but I also agree that there might be more. I will to focus on improving accessibility for the next days.
